domain/src/DomainForm.php

```diff
@@ -4,11 +4,65 @@
 
 use Drupal\Core\Entity\EntityForm;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\domain\DomainLoader;
+use Drupal\domain\DomainValidator;
+use Drupal\domain\DomainCreator;
+use Drupal\Core\Path\PathValidatorInterface;
 
 /**
  * Base form for domain edit forms.
  */
-class DomainForm extends EntityForm {
+class DomainForm extends EntityForm implements ContainerInjectionInterface {
+
+  /**
+   * @var \Drupal\domain\DomainLoader
+   */
+  protected $domainLoader;
+
+  /**
+   * @var \Drupal\domain\DomainValidator
+   */
+  protected $domainValidator;
+
+  /**
+   * @var \Drupal\domain\DomainCreator
+   */
+  protected $domainCreator;
+
+  /**
+   * @var PathValidatorInterface
+   */
+  protected $pathValidator;
+
+  /**
+   * DomainForm constructor.
+   *
+   * @param \Drupal\domain\DomainValidator $domainValidator
+   * @param \Drupal\domain\DomainLoader $domainLoader
+   * @param \Drupal\domain\DomainCreator $domainCreator
+   * @param PathValidatorInterface $pathValidator
+   */
+  public function __construct(DomainValidator $domainValidator, DomainLoader $domainLoader, DomainCreator $domainCreator, PathValidatorInterface $pathValidator) {
```

Comment:
all the variables in the method signature must be snake case.

Check this comment :-) https://www.drupal.org/node/2303963#comment-10111290